stylesheets/main.css

```diff
@@ -7,7 +7,7 @@
 
 	/* shuttle config */
 	--shuttle-logo: var(--shuttle-logo-jpl);
-	--rocket-fuel: 0;
+	--rocket-fuel: 11;
```

Comment:
You need to set this variable to 1. 
```suggestion
	--rocket-fuel: 1;
```

Reply:
Thanks, @gene-kranz